### ISSUE TITLE
Removing residue from combustion engines MC 1.11.2

### ISSUE THREAD

**Issue by jburkej1** [BuildCraft/BuildCraft#4069]
When I went to remove the residue from my combustion engines I found that when I used a bucket to drain it off, If I closed my inventory for a few minutes it went away. In itself it is a good thing but probably not what was expected. 

BuildCraft version: 7.99.8.2
Forge version: 8.0.99.99
Link to crash report or log: 
**Singleplaye**r or multiplayer: 
Steps to reproduce: 
Additional information:

**Comment by AEnterprise** [comment]
seems this slipped throught the cracks, has been fixed in 842e0a94b97f4936c7a3821297b96db838a87553 but might not have been backported to 1.11.2

**Comment by jburkej1** [comment]
no big issue for me as I moved back to 1.12.2 after seeing what was added in 1.11.x. I went from 1.8.9 to 1.12.2 and had a bit of culture shock at what had changed and been added. 
jb

**Comment by AEnterprise** [comment]
@AlexIIL do you want to backport this or just leave as is, don't think we have anything else for 1.11?

**Comment by AlexIIL** [comment]
Yes, this is a bug, and yes it needs fixing. I have no idea when I'll get around to it though.
